src/sentry/api/endpoints/organization_stats.py

```diff
@@ -90,9 +92,11 @@ def get(self, request, organization):
         if stat_model is None:
             raise ValueError("Invalid group: %s, stat: %s" % (group, stat))
 
-        data = tsdb.get_range(
-            model=stat_model, keys=keys, **self._parse_args(request, **query_kwargs)
-        )
+        try:
```

Comment:
Could we put the try/except in `_parse_args` prevent invalid 500s elsewhere its used?

Reply:
Yeah I could wrap each parameter in a try/catch. That would give us better error messages as well.

Reply:
Oh nice 👍 better error messages is a nice win too